.github/workflows/push_pr.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - main
       - master
```

Comment:
If it is tested at PR and renovate push time, I would remove `main`/`master`. I see no point in re-testing everything at `main` if we are going to autorelease it automatically. We are testing at pre-release time and that would stop the prerelease.

Reply:
Fair enough. Really good points here :)